### PR TITLE
Main api specs and workflow sync dev

### PR DESCRIPTION
**Pull request by storm20198**

### CODE REVIEW

File: api_specifications/hearing-api-0.1.0.yaml [hunk 1]
```diff
@@ -255,6 +269,11 @@ components:
         tenantId:
           type: string
           description: This is tenantId of the case
+        hearingId:
+          type: string
+          description: The formatted number given when a hearing gets created.
+          minLength: 2
```
Comment (atulgupta2024) [inline api_specifications/hearing-api-0.1.0.yaml:275]: from what I understand it is still a UUID. There is no hearing number like we have CNR For Case or Order ID for order. This is internal ID only

File: api_specifications/hearing-api-0.1.0.yaml [hunk 2]
```diff
@@ -279,13 +298,13 @@ components:
           description: workflow state being set as hearing status
           example: scheduled, in progress, in transcription, closed
         startTime:
-          type: string
-          format: date
+          type: integer
+          format: int64
           nullable: true
           description: This field holds information about start date time of hearing in epoch time.
         endTime:
```
Comment (atulgupta2024) [inline api_specifications/hearing-api-0.1.0.yaml:305]: do mention that it is epoch time in milliseconds

File: api_specifications/order-api-0.1.0.yaml [hunk 3]
```diff
@@ -241,33 +260,23 @@ components:
             description: application number the order is based on. Could be null if not based on any application
         hearingNumber:
           type: string
```
Comment (atulgupta2024) [inline api_specifications/order-api-0.1.0.yaml:262]: we may want to call this hearingId

**Review by atulgupta2024 — CHANGES_REQUESTED** [review]

File: api_specifications/hearing-api-0.1.0.yaml (hunk 1 above)
Comment (storm20198) [reply api_specifications/hearing-api-0.1.0.yaml:275]: This is generated via idgen , want me to remove it?

**Review by atulgupta2024 — APPROVED** [review]
reviewed. go ahead

**Pull request merged**